lms/services/blackboard_api/factory.py

```diff
@@ -16,7 +16,7 @@ def blackboard_api_client_factory(_context, request):
             redirect_uri=request.route_url("blackboard_api.oauth.callback"),
             http_service=request.find_service(name="http"),
             oauth_http_service=request.find_service(name="oauth_http"),
-            refresh_enabled=True,
+            refresh_enabled=False,
```

Comment:
_Disable_ the _backend_ refresh functionality.

Reply:
Is there a reason to keep `refresh_enabled` around, or will it make sense to just remove the capability entirely?